admin_manual/ai/app_context_chat.rst

```diff
@@ -69,15 +83,15 @@ Context chat will automatically load user data into the Vector DB using backgrou
 
 .. code-block::
 
-   occ background-job:worker 'OCA\ContextChat\BackgroundJobs\StorageCrawlJob'
+   set -e; while true; do sudo -u www-data occ background-job:worker -v -t 60 "OCA\ContextChat\BackgroundJobs\StorageCrawlJob"; done
```

Comment:
Shouldn't this have two backslashes? :thinking:

Reply:
dunno, works fine as is

Reply:
alright, nevermind then :)